lib/view_component/use_helpers.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module ViewComponent::UseHelpers
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def use_helpers(*args)
+      args.each do |helper_mtd|
+        class_eval(<<-RUBY, __FILE__, __LINE__ + 1)
+          def #{helper_mtd}(*args, &block)
+            raise HelpersCalledBeforeRenderError if view_context.nil?
+            __vc_original_view_context&.send(#{helper_mtd.inspect}, *args, &block) || view_context.send(#{helper_mtd.inspect}, *args, &block)
```

Comment:
```suggestion
            __vc_original_view_context&.send(#{helper_mtd.inspect}, *args, &block)
```

Can we avoid sneding to the non-original `view_context`?